src/effects/native/echoeffect.h

```diff
@@ -26,14 +26,14 @@ struct EchoGroupState {
             : delay_buf(mixxx::AudioSignal::kSamplingRateMax * kMaxDelaySeconds *
                         kChannelCount) {
```

Comment:
I'll increase it now that I've come up with a solution for the ridiculous memory usage in #1254.